src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Buttons } from '../../constants/common.constants';
+import classes from './SearchBar.module.scss';
+
+interface ISearchBarProps {
+  text: string;
+  onTextChange: (value: string) => void;
+}
+
+class SearchBar extends React.Component<ISearchBarProps, object> {
+  constructor(public props: ISearchBarProps) {
+    super(props);
+    this.handleTextChange = this.handleTextChange.bind(this);
```

Comment:
use arrow functions to avoid binding

Reply:
I after all can write down bind directly at transfer of function in onChange? corrected like this.